app/src/main/res/values/strings.xml

```diff
@@ -54,7 +54,7 @@
     <string name="last_name_optional">Last Name (Optional)</string>
 
     <!--eventyay-->
-    <string name="eventyay_logo">eventyay</string>
+    <string name="eventyay_logo">Eventyay</string>
```

Comment:
This should not be capitalised

Reply:
Ok, I will update these changes

Reply:
@GOVINDDIXIT Please use vector icons too